packages/editor/src/components/parts/program-start/StartPart.tsx

```diff
@@ -0,0 +1,41 @@
+import { IVY_SCRIPT_TYPES, ProgramStartData } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../editors';
+import { useValidations } from '../../../context';
+import { Input, useFieldset } from '../../widgets';
+import { PathFieldset } from '../common';
+import { useStartData } from './useStartData';
+import { Permission } from '../web-service/Permission';
+
+export function useProgramStartPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useStartData();
+  const compareData = (data: ProgramStartData) => [data.javaClass, data.permission];
+  const validation = useValidations(['javaClass']);
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return { name: 'Start', state, reset: { dirty, action: () => reset() }, content: <StartPart /> };
+}
+
+const StartPart = () => {
+  const { config, defaultConfig, update, updatePermission } = useStartData();
+
+  const javaClassField = useFieldset();
+
+  return (
+    <>
+      <PathFieldset label='Java Class' {...javaClassField.labelProps} path='javaClass'>
+        <Input
+          value={config.javaClass}
+          onChange={change => update('javaClass', change)}
+          type={IVY_SCRIPT_TYPES.STRING}

```

Comment:
the type is for a scriptinput, but has another meaning for a normal input field. So remove it.